.php-cs-fixer.dist.php

```diff
@@ -17,21 +17,64 @@
     ->setRules([
         '@PSR12' => true,
         'array_syntax' => ['syntax' => 'short'],
+        'backtick_to_shell_exec' => true,
         'braces' => [
             'allow_single_line_closure' => true,
             'allow_single_line_anonymous_class_with_empty_body' => true,
         ],
+        'class_definition' => ['single_line' => true],
         'concat_space' => ['spacing' => 'one'],
         'declare_strict_types' => true,
+        'elseif' => true,
+        'encoding' => true,
+        'ereg_to_preg' => true,
+        'explicit_string_variable' => true,
+        'fully_qualified_strict_types' => true,
         'function_typehint_space' => true,
+        'general_phpdoc_annotation_remove' => [
+            'annotations' => [
+                'author',
+                'package',
+                'subpackage',
+                'version',
+            ],
+        ],
+        'global_namespace_import' => [
+            'import_functions' => true,
+        ],
+        'include' => true,
+        'increment_style' => true,
         'list_syntax' => ['syntax' => 'short'],
+        'native_function_invocation' => [
+            'include' => ['@compiler_optimized'],
+        ],
+        'native_function_type_declaration_casing' => true,
+        'new_with_braces' => true,
+        'no_blank_lines_after_class_opening' => true,
+        'no_empty_comment' => true,
         'no_empty_phpdoc' => true,
         'no_empty_statement' => true,
+        'no_homoglyph_names' => true,
+        'no_leading_import_slash' => true,
         'no_leading_namespace_whitespace' => true,
+        'no_mixed_echo_print' => true,
```

Comment:
Shouldn't the value be either "print" or "echo"?

https://mlocati.github.io/php-cs-fixer-configurator/#version:3.8|fixer:no_mixed_echo_print